src/CommunityPatch/CommunityPatchSubModule.cs

```diff
@@ -26,7 +27,7 @@ public partial class CommunityPatchSubModule : MBSubModuleBase {
     internal static readonly LinkedList<Exception> RecordedUnhandledExceptions
       = new LinkedList<Exception>();
 
-    internal static readonly OptionsFile Options = new OptionsFile(nameof(CommunityPatch)+".txt");
+    internal static readonly OptionsStore Options = new OptionsFile(nameof(CommunityPatch) + ".txt");
```

Comment:
I'd prefer explicitly enforcing a file type/extension here.   Given they're all toml files as well.

Reply:
The other simple key-value-pair bannerlord config files are all .txt files, apart from that there's xml files.

Reply:
Ah I see the intention now, resolving.